Client/src/components/common/GoBackButton/GoBackButton.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useNavigate, useLocation } from 'react-router-dom';
+import styles from './GoBackButton.module.css';
+import whiteArrow from '../../../assets/images/whiteArrow.png';
+
+const GoBackButton = () => {
+  const navigate = useNavigate();
+  const location = useLocation();
+
+  const goBack = () => {
+    if (location.pathname.includes('edit')) {
+      navigate(-1);
+    } else if (location.pathname.includes('user')) {
+      navigate('/');
+    }
```

Comment:
make this function more dynamic.

like get the destination path as props. -> if null make it go back with navigate(-1)